cmd/cli/kubectl-kyverno/resource/resource.go

```diff
@@ -59,7 +59,8 @@ func YamlToUnstructured(resourceYaml []byte) (*unstructured.Unstructured, error)
 	return resource, nil
 }
 
-func GetResourceFromPath(fs billy.Filesystem, path string) ([]*unstructured.Unstructured, error) {
+// should be able to specify a single resource in a multi yaml file, dont error out when there are multiple resource
+func GetResourceFromPath(fs billy.Filesystem, path string, apiVersion, kind, resourceNamespace, resourceName string) (*unstructured.Unstructured, error) {
```

Comment:
Not sure i fully understand this, can you tell me more please ?

Reply:
These four paras `apiVersion, kind, resourceNamespace, resourceName string` are all exist in `TestResourceSpec`, can we re-use it instead of taking four string paras?